### PR TITLE
Pointsmodel

### PR DESCRIPTION
**Pull request by kathinka**
Added pointsmodel, and adjusted travel model som the user is not only a foreign key but required.

### CODE REVIEW

File: backend/models/points-model.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import mongoose from 'mongoose';
+
+const pointsSchema = new mongoose.Schema({
+  points: {
+    type: Number,
+    required: [true, "Points is required"],
+    min: [0, "Points cannot be negative"],
```
Comment (El1an3) [inline backend/models/points-model.js:7]: Good point, unless we want to "punish" users for bad choices. 😉

File: backend/models/travel-model.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ const travelSchema = new mongoose.Schema({
   user: {
     type: mongoose.Schema.Types.ObjectId,
     ref: 'User',
+    required: [true, "User is required"],
```
Comment (El1an3) [inline backend/models/travel-model.js:29]: 👍

**Review by El1an3 — APPROVED** [review]
Looks good to me. 
As backend is not my forte, I'd like someone else to have look too.

File: backend/models/points-model.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    ref: 'Travel',
+    required: [true, "Travel is required"],
+  },
+  date:{
+    type: Date,
+    required: [true, "Date is required"],
+  default: Date.now,
+  },
+});
+
```
Comment (vittoriamatteoli) [inline backend/models/points-model.js:25]: nice! it's all connected ✨

**Review by vittoriamatteoli — APPROVED** [review]

File: backend/models/points-model.js (hunk 1 above)
Comment (kathinka) [reply backend/models/points-model.js:7]: nah i think 0 is as bad as it should be haha

Comment (Yifan-858) [reply backend/models/points-model.js:7]: haha, good one!

**Review by Yifan-858 — APPROVED** [review]

**Pull request merged**